java/code/webapp/WEB-INF/pages/admin/multiorg/extgroup_detail.jsp

```diff
@@ -5,33 +5,31 @@
 <%@ taglib uri="http://rhn.redhat.com/tags/list" prefix="rl" %>
 
 <html:html >
-    <head>
+    <body>
         <script language="javascript" type="text/javascript">
-          function toggle_checkboxes() {
-            org_checkbox = document.getElementById('role_org_admin');
-            sat_checkbox = document.getElementById('role_satellite_admin');
-            role_checkboxes = document.getElementsByName('regular_role');
-            for (var i = 0, n = role_checkboxes.length; i < n; i++) {
-              if (org_checkbox.checked) {
-                role_checkboxes[i].checked = org_checkbox.checked;
-              }
-              role_checkboxes[i].disabled = org_checkbox.checked;
+            function toggle_checkboxes() {
+                org_checkbox = document.getElementById('role_org_admin');
+                sat_checkbox = document.getElementById('role_satellite_admin');
+                role_checkboxes = document.getElementsByName('regular_role');
+                for (var i = 0, n = role_checkboxes.length; i < n; i++) {
```

Comment:
Also, out of PR, but it could be nice to update some `for` loops using `forEach`
```
role_checkboxes.forEach(function(role_checkbox) {
    if (org_checkbox.checked) {
        role_checkbox.checked = org_checkbox.checked;
    }
    role_checkbox.disabled = org_checkbox.checked;
});
```

Reply:
This is really old code. When the page is refactored to react, we can properly improve and refactor this :)